kubernetes/apps/ai/llmkube/models/run-qwen36-decode-ab.sh

```
@@ -0,0 +1,137 @@
+POD="$(mise exec -- jq -er '[.items[] | select(.metadata.deletionTimestamp == null)] | if length == 1 then .[0].metadata.name else empty end' <<<"$PODS_JSON")"
+mise exec -- jq -e --arg pod "$POD" --arg expected_image "$EXPECTED_IMAGE" --arg expected_claim "$EXPECTED_CLAIM" --argjson expected_args "$EXPECTED_ARGS" '
+  [.items[] | select(.metadata.name == $pod)] as $selected
+  | if ($selected | length) != 1 then error("selected benchmark pod disappeared") else $selected[0] end
+  | if .status.phase != "Running" then error("benchmark pod is not Running") else . end
+  | if (any(.status.conditions[]?; .type == "Ready" and .status == "True") | not) then error("benchmark pod is not Ready") else . end
+  | if (.spec.containers | length) != 1 or .spec.containers[0].name != "sglang" then error("expected exactly one sglang container") else . end
+  | if .spec.containers[0].image != $expected_image then error("unexpected benchmark image") else . end
+  | if .spec.containers[0].command != ["python3", "-m", "sglang.launch_server"] then error("unexpected benchmark command") else . end
+  | if .spec.containers[0].args != $expected_args then error("unexpected benchmark args") else . end
+  | if ([.spec.volumes[]? | select(.name == "triton-cache") | .persistentVolumeClaim.claimName] != [$expected_claim]) then error("unexpected Triton PVC") else . end
+' <<<"$PODS_JSON" >/dev/null
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)